server/middleware/auth.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken')
+require('dotenv').config();
```

Comment:
NO! You already told the server to use environment variables to the server in the index.js remember?
You don't to to call it here too!